lhotse/augmentation/utils.py

```diff
@@ -69,3 +74,134 @@ def convolve1d(signal: torch.Tensor, kernel: torch.Tensor) -> torch.Tensor:
     result = irfft(f_result, n=fast_ftt_size)
 
     return result[:padded_size]
+
+
+# The following is taken from: https://github.com/yluo42/FRA-RIR/blob/main/FRA-RIR.py
+def generate_fast_random_rir(
```

Comment:
If it turns out to be too slow, I would check if it's faster with pure `numpy` that with CPU `torch`, that often turned out to be true for me. If it's too much hassle it's OK to leave as is.